src/backend/distributed/transaction/transaction_management.c

```diff
@@ -215,6 +215,8 @@ CoordinatedTransactionCallback(XactEvent event, void *arg)
 
 		case XACT_EVENT_PRE_COMMIT:
 		{
+			bool using2PC = MultiShardCommitProtocol == COMMIT_PROTOCOL_2PC;
```

Comment:
This name seems a bit misleading, we might not be using 2PC here even if MultiShardCommitProtocol == COMMIT_PROTOCOL_2PC.